### PR TITLE
fixed pylint, formatted python files iwth cblack on localhost

### PR DESCRIPTION
**Pull request by iainwo**
Fixed linter check for indentation.
Fixes geohot/tinygrad/#198 using @Liamdoult and @gamwe6's suggestions.
I reindent the code using: https://github.com/python/cpython/blob/master/Tools/scripts/reindent.py by swapping out all the 4s for 2s and running locally. Tests pass but we're probably lacking some code coverage.

Here is the output of CI.
```
Run python -m pylint --disable=all -e W0311 --jobs=0 --indent-string='  ' **/*.py
  python -m pylint --disable=all -e W0311 --jobs=0 --indent-string='  ' **/*.py
  shell: /bin/bash -e {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.8.6/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.8.6/x64/lib

------------------------------------
Your code has been rated at 10.00/10
```

### CODE REVIEW

File: ane/1_build/coreml_ane.py [hunk 1]
```diff
@@ -3,24 +3,34 @@
...
+builder.add_inner_product(
+  name="ip_layer",
+  W=weights,
+  b=None,
+  input_channels=3,
+  output_channels=3,
+  has_bias=False,
+  input_name="image",
+  output_name="med",
+)
```
Comment (Liamdoult) [inline ane/1_build/coreml_ane.py:22]: These multi-line conversions will need to be reverted to one line.

**Review by Liamdoult — COMMENTED** [review]
P.S. For future reference, you can keep one PR. No need to manually squish. The PR will be squashed when merged anyway.

**Comment by Liamdoult** [comment]
Closes #198

File: ane/1_build/coreml_ane.py (hunk 1 above)
Comment (iainwo) [reply ane/1_build/coreml_ane.py:22]: thanks @Liamdoult saw black expanded everything

Comment (iainwo) [reply ane/1_build/coreml_ane.py:22]: gonna try and only run the formatter to only change indent

Comment (iainwo) [reply ane/1_build/coreml_ane.py]: used reindent script from cython to only indent (added description in the top of the PR)

**Pull request merged**